guacamole/src/main/frontend/src/app/settings/directives/guacSettingsPreferences.js

```diff
@@ -78,6 +101,26 @@ angular.module('settings').directive('guacSettingsPreferences', [function guacSe
              */
             $scope.preferences = preferenceService.preferences;
 
+            /**
+             * All available user attributes, as a mapping of form name to form
+             * object. The form object contains a name, as well as a Map of fields.
+             *
+             * The Map type is used here to maintain form/name uniqueness, as well as
+             * insertion order, to ensure a consistent UI experience.
+             *
+             * @type Map<String, Object>
+             */
+            $scope.attributeMap = new Map();
```

Comment:
It does not. Is that a problem? This usage of the `Map` data structure looks pretty well supported, including in IE11.

Reply:
Yep, because it's not supported in IE10, which we have not dropped support for. That said, I'm surprised that it doesn't get polyfilled during compilation now that Closure Compiler is back.

Reply:
Ah, ok. I'll find another way.

Reply:
And yeah, it is strange that it's not polyfilled - it sure sounds like it _should_ be, based on the documentation: https://github.com/google/closure-compiler/wiki/Polyfills